bittensor/commands/register.py

```diff
@@ -264,7 +264,7 @@ def run(cli):
     @staticmethod
     def add_args(parser: argparse.ArgumentParser):
         run_faucet_parser = parser.add_parser(
-            "faucet", help="""Register a wallet to a network."""
+            "faucet", help="""Perform PoW to receieve test TAO in your wallet."""
```

Comment:
Why does this command perform PoW ? Sorry for my ignorance here

Reply:
It's simply to run the faucet with a very low difficulty so takes ~15-30 sec.  So people don't spam and get instant tokens.

Reply:
TIL :)